mrtt-ui/src/components/SiteInterventions/SiteInterventions.js

```diff
@@ -141,7 +141,7 @@ function SiteInterventionsForm() {
   } = useFieldArray({ name: 'whichStakeholdersInvolved', control })
 
   const { siteId } = useParams()
-  const apiAnswersUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/registration_answers`
+  const interventionFormsUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/intervention_answers`
```

Comment:
this is going to screw up the `successCallback` and will be addressed in upcoming prs

Reply:
(I wont merge until all the prs for this are approved and its bug free in totality, just need to break up for my own sanity)